pyke/prf_photometry.py

```diff
@@ -0,0 +1,407 @@
+from abc import abstractmethod
+import math
+import scipy
+import numpy as np
+import tqdm
+import sys
+from astropy.io import fits as pyfits
+from oktopus.posterior import PoissonPosterior
+from .utils import channel_to_module_output, plot_image
+
+if sys.version_info[0] == 2:
+    from inspect import getargspec
+    def _get_number_of_arguments(func):
+        return len(getargspec(func).args)
+
+else:
+    from inspect import signature
+    def _get_number_of_arguments(func):
+        return len(signature(func).parameters)
+
+
+__all__ = ['PRFPhotometry', 'SceneModel', 'KeplerPRF', 'SimpleKeplerPRF', 'get_initial_guesses']
+
+
+class PRFPhotometry(object):
+    """
+    This class performs PRF Photometry on TPF-like files.
+
+    Attributes
+    ----------
+    scene_model : instance of SceneModel
+        Model which will be fit to the data
+    priors : instance of oktopus.JointPrior
+        Priors on the parameters that will be estimated
+    loss_function : subclass of oktopus.LossFunction
+        Noise distribution associated with each random measurement
+    """
```

Comment:
Let's add an `Example` section to this class docstring which shows how to instantiate the class with a simple `scene_model` and `priors`?